language/spec/language_samples_spec.rb

```diff
@@ -45,20 +45,54 @@ def capture &block
     $stdout = real_stdout
   end
 
-  example "sentiment from text" do
+  example "document sentiment from text" do
     positive_text = "Happy love it. I am glad, pleased, and delighted."
     negative_text = "I hate it. I am mad, annoyed, and irritated."
 
     expect {
       sentiment_from_text project_id: @project_id, text_content: positive_text
-    }.to output(/^\d\.\d+ \(\d\.\d+\)/).to_stdout
+    }.to output(/Overall document sentiment: \(\d\.\d+\)$/).to_stdout
 
     expect {
       sentiment_from_text project_id: @project_id, text_content: negative_text
-    }.to output(/^-\d\.\d+ \(\d\.\d+\)/).to_stdout
+    }.to output(/Overall document sentiment: \(-\d\.\d+\)$/).to_stdout
+
+  end
+
+  example "sentence-level sentiment from text" do
+    positive_text = "Happy love it. I am glad, pleased, and delighted."
+    negative_text = "I hate it. I am mad, annoyed, and irritated."
+
+    expect {
+      sentiment_from_text project_id: @project_id, text_content: positive_text
+    }.to output(/Sentence .*\n^.*\(\d\.\d+\)$\n^.*\(\d\.\d+\)$/).to_stdout
+
+    expect {
+      sentiment_from_text project_id: @project_id, text_content: negative_text
+    }.to output(/Sentence .*\n^.*\(-\d\.\d+\)$\n^.*\(-\d\.\d+\)$/).to_stdout
```

Comment:
This test should expect output to include information understood from `text_content`